typescript/infra/src/core/govern.ts

```diff
@@ -0,0 +1,109 @@
+import { ethers } from 'ethers';
+
+import {
+  AbacusCoreChecker,
+  ChainMap,
+  ChainName,
+  CoreViolationType,
+  OwnerViolation,
+  ValidatorViolation,
+  ValidatorViolationType,
+  ViolationType,
+  objMap,
+} from '@abacus-network/sdk';
+import { types } from '@abacus-network/utils';
+
+interface Call {
+  to: types.Address;
+  data: ethers.utils.BytesLike;
+}
```

Comment:
we have something similar here https://github.com/abacus-network/abacus-monorepo/blob/165f60e6226c4850ae64a003d8e638e28c89d17d/typescript/utils/src/types.ts#L14
`BytesLike` looks nicer